src/lib/Dashboard/TabInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace EzSystems\HybridPlatformUi\Dashboard;
+
+interface TabInterface
```

Comment:
This is the same as `SectionInterface`. They are small, but could we reduce duplication by having a common top level interface, or a `Renderable` trait, maybe?
Also, missing method doc.